package.json

```diff
@@ -30,6 +31,7 @@
 		"cypress:update-snapshots": "cypress run-ct --env type=base --config screenshotsFolder=cypress/snapshots/base"
 	},
 	"main": "dist/ncvuecomponents.js",
```

Comment:
Would this not need to point to install to keep previous behavior to expose a vue plugin? Otherwise we may add some documentation for that to the readme maybe.

Reply:
Doc updated